src/Http/Controllers/CategoryAPIController.php

```diff
@@ -49,7 +49,7 @@ public function index(CategoryListRequest $request): JsonResponse
             $request->get('limit')
         );
 
-        return CategoryResource::collection($categories)->response();
+        return $this->sendResponseForResource(CategoryResource::collection($categories), "Categories retrieved successfully");
```

Comment:
@victazzz @tomekgubernat please check if this is not messing with front/admin now